src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -49,6 +49,7 @@ object ErrorAggregator {
     .add[Int]("PERMISSIONS_SQL_CORRUPTED")
     .add[Int]("DEFECTIVE_PERMISSIONS_SQL_REMOVED")
     .add[Int]("SLOW_SCRIPT_NOTICE_COUNT")
+    .add[Int]("SLOW_SCRIPT_PAGE_COUNT")
```

Comment:
I'm not sure I follow you. Once a metric is there, it will added to the schema and it will be retrieved by `getCountHistogramValue`, right?